CMakeLists.txt

```diff
@@ -170,6 +170,35 @@ install(
     PATTERN "internal" EXCLUDE
 )
 
+if(${PYLON_DOWNLOADED})
```

Comment:
I would create a separate package instead of merging this into the pylon_camera package